packages/desktop/views/dashboard/campaigns/views/CampaignDetailsView.svelte

```diff
@@ -1,8 +1,79 @@
     import Leaderboard from '../components/Leaderboard.svelte'
     import { campaignLeaderboards, addCampaignLeaderboard, selectedCampaign } from '@contexts/campaigns'
-    import { TideApi } from '@core/tide/apis'
+    import UserPositionCard from '../components/UserPositionCard.svelte'
+    import { selectedAccount } from '@core/account/stores'
+    import { getAddressFromAccountForNetwork } from '@core/account'
+
+    const userNft: Nft = {
+        id: '0x9cb0f842bb6f827806f46cbbf62a494e6779bd08:1',
+        type: MimeType.ImagePng,
+        networkId: SupportedNetworkId.ShimmerEvm,
+        name: 'MafiaBird #1',
+        isLoaded: true,
+        isSpendable: false,
+        standard: NftStandard.Erc721,
+        uri: 'https://tideprotocol.infura-ipfs.io/ipfs/QmZHMqGJ69SCofyoXtKaXxrLnoKBbGL92aC5NC79sk9Aks',
+        contractMetadata: {
+            standard: NftStandard.Erc721,
+            address: '0x9cb0f842bb6f827806f46cbbf62a494e6779bd08',
+            name: 'MafiaBirds',
+            symbol: 'MB',
+        },
+        tokenId: '1',
+        metadata: {
+            type: MimeType.ImagePng,
+            name: 'MafiaBird #1',
+            description:
+                'Are you ready to become a made bird and dive into The Magpie Mafia? 🐦🗡️ Enter the gang, climb your way up thru the ranks of a fun-oriented mafia-style organization, and be rewarded for your activity. 🔥🫡  By joining, you will earn your own nontransferable Magpie Mafia NFT that will level up based on your activity while acting as your way to access future Magpie Mafia functions, track your reputation, and more! 💣From here onwards, each time you’d like to engage in a campaign, join an event, or track your reputation, just show that you’re a member of The Magpie Mafia and join in on all the fun. 💰The beak speaks, the words of The Magpie Mafia carries with it authority and respect. 🪖',
+            image: 'https://tideprotocol.infura-ipfs.io/ipfs/QmZHMqGJ69SCofyoXtKaXxrLnoKBbGL92aC5NC79sk9Aks',
+            attributes: [
+                {
+                    trait_type: 'Body',
+                    value: 'Rocco',
+                },
+                {
+                    trait_type: 'Clothes',
+                    value: 'Harlem',
+                },
+                {
+                    trait_type: 'Background',
+                    value: 'Empire State Building',
+                },
+                {
+                    trait_type: 'Role',
+                    value: 'Ally',
+                },
+                {
+                    trait_type: 'Weapon',
+                    value: 'Bird\'s Knuckles',
+                },
+                {
+                    trait_type: 'XP',
+                    value: '65',
+                },
+            ],
+        },
+    }
+
+    let userPosition: ITideLeaderboardItem | undefined = undefined
+    $: userPosition = $campaignLeaderboards[$selectedCampaign.projectId]?.[$selectedCampaign.id]?.find(
```

Comment:
this wont work because the leaderboard is only giving us top 10, and the user may not be in top ten, we need an action to get leaderboard filtered by address:

https://api-prod.tideprotocol.xyz/public/project/490/leaderboard?cids=609dc557-1ca3-4f6d-9ff7-b655a5584653&by=ADDRESS&search=0x7d06dE4aE53Ef27Fff2B34731C97bb44FD27D9E6